### PR TITLE
Bl 465 update product doc repository to include homepage launch details

### PR DESCRIPTION
**Pull request by Rebecca-Stone**
## Description

Make updates to [the product documentation repository](https://github.com/BloomTech-Labs/coder-heroes-docs) based on the changes in this epic.

#### Video Link

[Loom Video](https://www.loom.com/share/f566d6b893964be28ea51ddd8bcdbfb6)

#### Trello Link

https://bloomtechlabs.atlassian.net/browse/BL-465?atlOrigin=eyJpIjoiYzRkMDEwZDhlOWQxNDE0NDhkMjc4YmFjYTdmMmMxOTQiLCJwIjoiaiJ9

## Checklist:

- [X] My design/diagram follows the style guidelines of this project
- [X] My docs are organized within an appropriate folder structure in our repo
- [X] Pull request description clearly describes docs made & motivations for said additions

### CODE REVIEW

**Review by leewp8 — APPROVED** [review]
Nice work and great job organizing all of this. In my opinion, this stuff is just as important to the project as the hard coding.

**Comment by Rlauckern** [comment]
Great job organizing and i love the idea of the release manager adding a documents folder for everyone to read and add to.

File: homepage_launch.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ When A consumer visits the CoderHeroes website, they should be able to find info
 _Assignee: Conrad Klek_ - _Co-owner: James Fincher_
 
 [Figma](https://www.figma.com/file/7VBR8i7t3nbdmUp15ELqCG/CoderHeroes-Figma?node-id=0%3A1)
+
 **This ticket is only to update the assets in Figma. Once the stakeholder approves of the changes, then we can apply the updated fonts to the entire mockup designs as well as on the web app itself.**
 
 During our stakeholder meeting with Brianne on June 30, she pointed out that the stylized font used in headers seemed a bit hard to read. She stated that she really loved the [Girls Who Code](https://girlswhocode.com/) website, and they use Roboto as the font face across the entire site.
```
Comment (Rlauckern) [inline homepage_launch.md:85]: Great job organizing and i love your suggestion of having the release manager add a documents folder for people to read and add to.

**Review by Rlauckern — APPROVED** [review]
Great job organizing and i love your suggestion of having the release manager add a documents folder for people to read and add to.

File: homepage_launch.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+- About:
+  - About CoderHeroes
+  - Terms & Conditions + Privacy Policy (these can both follow the same template)
+  - Press Inquiries
+- Services:
+  - FAQ
+  - Site Map
+
+---
```
Comment (dirkknibbe) [inline homepage_launch.md:110]: Very comprehensive and detailed, great job!

**Review by dirkknibbe — APPROVED** [review]

**Review by laurensruiz — APPROVED** [review]
Great job organizing everything! This will help a lot of students who might be seeing the repo for the first time.

**Review by AnthonyLayne — APPROVED** [review]
Your organization is great, and I love how you made each task name clearly visible, it really helps delineate them from one another.

**Review by techlando — APPROVED** [review]
Well done! Extremely organized and just as important as any code!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
 # coder-heroes-docs
+
 Documentation for the Coder Heroes LMS Labs project
+
+[BL-461 CoderHeros homepage launch details](/homepage_launch.md#bl-461---coderheros-homepage-launch-details)
+
+- [BL-462](/homepage_launch.md#bl-462---fe-i-want-to-find-information-that-will-tell-me-about-coderheroes) - FE: I want to find information that will tell me about CoderHeroes
```
Comment (dirkknibbe) [inline README.md:7]: Love the links to the exact code described!

**Review by dirkknibbe — APPROVED** [review]

**Pull request merged**